source/configure/user-management-configuration-settings.rst

```diff
@@ -39,30 +39,49 @@ Getting people set up with a Mattermost account is typically something that syst
 +----------------------------------------------------------------+-------------------------------------------------------------+
 
 Identify a User's ID
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~
+
 Users can be specified in Mattermost by username or user ID. Usernames automatically resolve when a match is detected.
-System admins can identify a user's ID using the Mattermost API or mmctl. 
+System admins can identify a user's ID using the Mattermost API or mmctl.
+
+Use the System Console
+^^^^^^^^^^^^^^^^^^^^^^
+
+Go to **System Console > User Management > Users** to access all user accounts. Select a user to review their ID. 
+
+.. tip::
+  
+  From Mattermost v9.6, you can:
+  
+  - Customize this page by showing or hiding user details, including email address, member duration, last login, activity, or post, number of days active, and number of messages posted. You can also control how many user records display on the page at a time.
```

Comment:
The stats fields like messaged posts or days active are exclusive to Postgres. Not sure if that needs to be called out.
They work in MySQL, but only via the export functionality.